spec/DSPX-4221.md

```
@@ -0,0 +1,40 @@
+    - opentdf/java-sdk:DSPX-4221-pq-sessions
+    - opentdf/web-sdk:DSPX-4221-pq-sessions
+prs: []
+created: 2026-07-31T00:00:00Z
+updated: 2026-07-31T00:00:00Z
+jira_priority: Medium
+---
+
+
+# Session Keys should support ML-KEM
+
+## Summary
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)